app/react/Metadata/components/RelationshipLink.tsx

```diff
@@ -27,26 +27,25 @@ const connector = connect(mapStateToProps, mapDispatchToProps);
 
 type mappedProps = ConnectedProps<typeof connector> & RelationshipLinkProps;
 
-const RelationshipLink = ({ prop: propVal, selectConnection }: mappedProps) =>
-  propVal.value.map((prop: any) => {
-    if (prop.relatedEntity) {
-      return (
-        <a
-          onClick={() => selectConnection(prop.relatedEntity)}
-          key={prop.url}
-          style={{ cursor: 'pointer' }}
-        >
-          {prop.value}
-        </a>
-      );
-    }
+const RelationshipLink = ({ propValue: propVal, selectConnection }: mappedProps) => {
+  if (propVal.relatedEntity) {
     return (
-      <I18NLink key={prop.url} to={prop.url}>
-        {prop.icon && <Icon className="item-icon" data={prop.icon} />}
-        {prop.value}
-      </I18NLink>
+      <a
```

Comment:
This is not correct.  `a` tags should be links.  If you are only looking for an onClick event, this should be a button.